prow/cmd/deck/template/job-history.html

```diff
@@ -0,0 +1,55 @@
+{{define "title"}}Job History: {{.Name}}{{end}}
+{{define "content"}}
+<style>
+  .run-success {
+    background-color: rgba(0, 255, 0, 0.3);
+  }
+  .run-failure {
+    background-color: rgba(255, 0, 0, 0.3);
+  }
+  .run-pending {
+    background-color: rgba(255, 255, 0, 0.3);
+  }
+</style>
```

Comment:
(matching the earlier suggestion)

```suggestion
</style>
{{end}}
{{define "content"}}
```